build.gradle.kts

```diff
@@ -41,6 +43,18 @@ ktlint {
     enableExperimentalRules.set(true)
 }
 
+open class SimpleCliTask : org.jetbrains.intellij.tasks.RunIdeTask() {
```

Comment:
Probably you should name it `KotlinAnalysisCliTask` since you have "kotlin-analysis" param for it